examples/python/code_graph_example.py

```diff
@@ -0,0 +1,13 @@
+import argparse
+import asyncio
+from evals.eval_swe_bench import run_code_graph_pipeline
+
+async def main(repo_path):
+    async for result in await run_code_graph_pipeline(repo_path):
+        print(result)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("repo_path", type=str, help="Path to the repository")
```

Comment:
I prefer arguments starting with "--", but it doesn't really matter while there is only one argument.

Reply:
Absolutely, it's neater. Improved it